options/options.html

```
@@ -284,6 +291,11 @@ <h3>保存目標</h3>
                 <option value="">選擇資料來源...</option>
               </select>
 
+              <div class="input-wrapper">
+                <input type="text" id="database-id" placeholder="輸入 Page ID 或 Database ID" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)